packages/mgt-chat/src/components/ChatListItem/ChatListItem.tsx

```diff
@@ -128,9 +128,14 @@ export const ChatListItem = ({ chat, myId, isSelected, isRead }: IMgtChatListIte
 
   // manage the internal state of the chat
   const [chatInternal, setChatInternal] = useState(chat);
-  const [read, setRead] = useState<boolean>(isRead);
+  const [read, setRead] = useState<boolean>();
   const cache = new LastReadCache();
 
+  // when isRead changes, setRead to match
+  useEffect(() => {
+    setRead(isRead);
+  }, [isRead]);
```

Comment:
Since GraphChatThread is a prop to ChatListItem, we don't need state here. When it changes, it will trigger a re-render.